### PR TITLE
feat:Product page added

### PR DESCRIPTION
**Pull request by ArpitPatni**
Please review my products page.

### CODE REVIEW

**Review by arulk8 — APPROVED** [review]
Looks good to go.

File: product.html [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      </div>
+      <div class="filter-cat">
+        <span><b>Rating</b></span>
+        <div class="filter-item">
+          <input type="checkbox">
+          4star & above
+        </div>
+        <div class="filter-item">
+          <input type="checkbox">
+          4star & above
```
Comment (arulk8) [inline product.html]: already 4star & above is there.

**Review by Balakumar3018 — APPROVED** [review]
Everything looks good to me.If i click product in the navigation the navigation icons collapsing.

**Review by sohan9819 — CHANGES_REQUESTED** [review]
Fix the navbar and bring the icons in one line . ( Try by decreasing the margins of your cart and heart icons )
Use radio inputs instead of check boxes for ratings  I think that will make more sense .
(suggestion) Try to put the "Add to cart" button at the bottom of the card container it will look much better . 
(suggestion) Add ```cursor : pointer``` to the heart icon so that user gets to know its kind of an input .

**Pull request merged**